test/Npgsql.Tests/DistributedTransactionTests.cs

```diff
@@ -546,7 +546,7 @@ public TransactionEvent(string message)
 
         #region Setup
 
-        NpgsqlConnection _controlConn;
+        NpgsqlConnection _controlConn = default!;
```

Comment:
This is a elegant way to cover the fact that we actually use `_controlConn = null!;` (Translates to: "Believe me, this `null` isn't a `null`).
I know... it gets initialized by `OneTimeSetup()`.
But again I don't deem the bang necessary here.
We could just use a bang in the one place where the compiler detects an uninitialized `_controlConn` which would translate to "Believe me, I know that `_controlConn` has been initialized by now.

Reply:
Both are possible - I don't feel it's very important, especially in a test class such as this.

But I'm thinking about this way: [OneTimeSetup] acts pretty much like a constructor; if it really were a constructor we wouldn't need to worry about this, and _controlConn would definitely be non-nullable. So to me it seems a tiny bit preferable to write the code as if we have a real constructor...